<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QWidget" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="AppSettings">
     <property name="title">
      <string>应用设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="source_l">
        <property name="text">
         <string>壁纸来源：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QComboBox" name="source">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="updateTime_l">
        <property name="text">
         <string>更新间隔：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QSpinBox" name="updateTime">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string>(分钟)</string>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1440</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="earthSize_l">
        <property name="text">
         <string>地球大小：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QSpinBox" name="earthSize">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="minimum">
         <number>20</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="wallpaperDir_l">
        <property name="text">
         <string>壁纸文件夹：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="wallpaperDir"/>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="selectDir">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="wallpaperFile_l">
        <property name="text">
         <string>壁纸文件：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="wallpaperFile"/>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="selectFile">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="apikey_l">
        <property name="text">
         <string>apiKey：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="3">
       <widget class="QLineEdit" name="apikey">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>使用NSFW需要提供apikey</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="searchkey_l">
        <property name="text">
         <string>关键词：</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="3">
       <widget class="QLineEdit" name="searchkey"/>
      </item>
      <item row="7" column="0" colspan="4">
       <layout class="QHBoxLayout" name="categoriesGroup">
        <item>
         <widget class="QLabel" name="categories_l">
          <property name="text">
           <string>分类：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="General">
          <property name="text">
           <string>General</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="Anime">
          <property name="text">
           <string>Anime</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="People">
          <property name="text">
           <string>People</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0" colspan="4">
       <layout class="QHBoxLayout" name="purityGroup">
        <item>
         <widget class="QLabel" name="purity_l">
          <property name="text">
           <string>分级：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="SFW">
          <property name="styleSheet">
           <string notr="true">font: 700;
color: #447744;</string>
          </property>
          <property name="text">
           <string>SFW</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="Sketchy">
          <property name="styleSheet">
           <string notr="true">font: 700;
color: #777744;</string>
          </property>
          <property name="text">
           <string>Sketchy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="NSFW">
          <property name="styleSheet">
           <string notr="true">font: 700;
color: #774444;</string>
          </property>
          <property name="text">
           <string>NSFW</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="sorting_l">
        <property name="text">
         <string>随机列表：</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="3">
       <widget class="QComboBox" name="sorting">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="color_l">
        <property name="text">
         <string>颜色：</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1" colspan="3">
       <widget class="QComboBox" name="color">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="atleast_l">
        <property name="text">
         <string>最小尺寸：</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1" colspan="3">
       <widget class="QComboBox" name="atleast"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="SystemSettings">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>系统设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="proxyTypeGroup">
        <item>
         <widget class="QLabel" name="proxy_l">
          <property name="text">
           <string>网络代理：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="proxyNone">
          <property name="text">
           <string>无</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="proxyHttp">
          <property name="text">
           <string>http</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="proxySocks">
          <property name="text">
           <string>socks</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="addGroup">
        <item>
         <widget class="QLabel" name="add_l">
          <property name="text">
           <string>地址：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="addEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="port_l">
          <property name="text">
           <string>:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="portEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="clearCache">
        <property name="text">
         <string>删除缓存</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="scripts_l">
        <property name="text">
         <string>外部脚本：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="scripts">
        <property name="placeholderText">
         <string>正常使用不需要填写，用来设置壁纸的脚本</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="applySpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>57</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="applyBtn">
     <property name="text">
      <string>应用</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="closeBtn">
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
